syft/generic/utils.py

```diff
@@ -15,3 +19,32 @@ def __call__(self, *args):
     def __missing__(self, key):
         result = self[key] = self.func(*key)
         return result
+
+
+def allow_command(func):
+    func_name = f"{func.__module__}.{func.__name__}"
+    allowed_commands.update({func_name})
+    return func
+
+
+def remote(func, location):
+    command_name = f"{func.__module__}.{func.__name__}"
+
+    worker = sy.local_worker
+
+    if isinstance(location, str):
+        location = worker.get_worker(location)
+
+    def remote_exec(*args, return_value=False, return_arity=1, multiprocessing=False, **kwargs):
```

Comment:
what if the function returns more than a value? (`return_arity=1`)

Reply:
If so, the existing mechanism with the ResponseSignatureError is used!
The idea is to have a clean way to avoid this if you already know you're gonna return `n` values

Reply:
And can't this be parameterized? If I want to execute a command that returns (True, 5) for example, can I use the remote decorator? It would be great if this can be passed as a parameter to the decorator with 1 being the default